vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/TouchSteps.java

```diff
@@ -123,19 +123,17 @@ public void swipeToElement(SwipeDirection direction, Locator locator, Duration s
         locator.getSearchParameters().setWaitForElement(false);
         Supplier<Rectangle> swipeArea = createSwipeArea();
         List<WebElement> elements = new ArrayList<>(searchActions.findElements(locator));
-        if (elements.isEmpty())
+        Rectangle swipeAreaRectangle = swipeArea.get();
+        touchActions.swipeUntil(direction, swipeDuration, swipeAreaRectangle, () ->
```

Comment:
having this logic at least one swipe will be performed, but it might be not needed, or even more it might move the element outside screen

Reply:
You are right, updated